### PR TITLE
EWPP-1787: Add status description field.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: modules/oe_content_event/src/EventNodeWrapper.php [hunk 1]
```diff
@@ -97,6 +97,27 @@ public function getEndDate(): DrupalDateTime {
     return $this->entity->get('oe_event_dates')->end_date;
   }
 
+  /**
```
Comment (yenyasinn) [inline modules/oe_content_event/src/EventNodeWrapper.php]: Methods should be tested in modules/oe_content_event/tests/src/Kernel/EventNodeWrapperTest.php

File: modules/oe_content_event/config/install/core.entity_form_display.node.oe_event.default.yml [hunk 2]
```diff
@@ -160,37 +161,27 @@ content:
...
   created:
     type: datetime_timestamp
-    weight: 25
+    weight: 26
     region: content
     settings: {  }
     third_party_settings: {  }
-  oe_event_programme:
-    weight: 39
+  field_oe_event_status_descriptio:
```
Comment (nagyad) [inline modules/oe_content_event/config/install/core.entity_form_display.node.oe_event.default.yml]: This needs to be fixed to oe_event_status_description.

File: modules/oe_content_event/config/post_updates/20002_update_form_display/core.entity_form_display.node.oe_event.default.yml [hunk 3]
```diff
@@ -160,37 +161,27 @@ content:
...
   created:
     type: datetime_timestamp
-    weight: 25
+    weight: 26
     region: content
     settings: {  }
     third_party_settings: {  }
-  oe_event_programme:
-    weight: 39
+  field_oe_event_status_descriptio:
```
Comment (nagyad) [inline modules/oe_content_event/config/post_updates/20002_update_form_display/core.entity_form_display.node.oe_event.default.yml]: Same here.

**Review by nagyad — CHANGES_REQUESTED** [review]

File: modules/oe_content_event/config/install/core.entity_form_display.node.oe_event.default.yml (hunk 2 above)
Comment (22Alexandra) [reply modules/oe_content_event/config/install/core.entity_form_display.node.oe_event.default.yml]: It actually needs to be removed cause it's duplicated. The proper one is lower.

**Review by nagyad — DISMISSED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**